Test.cs

```diff
@@ -12,7 +14,12 @@ public static long calculate(long number)
                 throw new Exception("Number must be non-negative.");
             }
 
-            return number == 0 ? 1 : number * Calculate(number + 1);
+            return number == 0 ? 1 : number * Calculate(number - 1);
+        }
+
+         public static string GetBranchNama()
+        {
```

Comment:
The method name `GetBranchNama` contains a typo. It should be corrected to `GetBranchName` for clarity and consistency.